queries/javascript/highlights.scm

```diff
@@ -184,6 +184,7 @@
 
 (binary_expression "/" @operator)
 (ternary_expression ["?" ":"] @operator)
+(unary_expression) @punctuation.special
```

Comment:
Is it not an operator ?

Reply:
https://tc39.es/ecma262/#sec-unary-operators
 
Looks like they would be operators.